blosc/eframe.c

```diff
@@ -55,12 +55,12 @@ void* eframe_append_chunk(blosc2_frame *frame, uint8_t *chunk, int32_t nchunk, i
 
 
 /*Get chunk from extended frame. */
-int eframe_get_chunk(blosc2_frame *frame, int64_t nchunk, uint8_t **chunk, bool *needs_free){
+int eframe_get_chunk(blosc2_frame* frame, int64_t nchunk, uint8_t** chunk, bool* needs_free){

```

Comment:
`nchunk` is always `int32_t`.  Please fix this here in in some possible other places.

Reply:
Well, in the case of extended frames, nchunk can certainly be more than 32-bit.  So, maybe it is a good idea to promote nchunk/nchunks to be `int64_t` in (extended) frames.